### ISSUE TITLE
Enhancement request: Reporting likelihoods in Bayesian tests

### ISSUE THREAD

**Issue by fplatz** [jasp-stats/jasp-desktop#2507]
* Enhancement: Reporting models' likelihoods in Bayesian tests in output tables
* Purpose: For the purpose of calculating posterior model odds (with own prior probabilities) as well as for teaching statistics for instance, it would be great to have the likelihoods of every model in the output table. It might be the case that some won't need this information, a possible solution might be the implementation of an option in the tests' menu like "tick here for a report of models' likelihoods". 
Thanks a lot! 

Best,
Friedrich

**Comment by EJWagenmakers** [comment]
You mean the marginal likelihood, right? For which tests in particular?
E.J.

**Comment by fplatz** [comment]
Yes, the marginal likelihood (excuse my sloppy terminology). First of all in the ANOVA, but the t-test as well as the binomial test would be great (however, for the last one, it would not be as essential as for the other test, due to the possibility of calculating by hand). 

Best,
Friedrich

**Comment by AlexanderLyNL** [comment]
Dear Friedrich,

The marginal likelihoods of each model are not necessary to calculate the posterior model odds. They can be calculated as follows

    P(M1 | d) / P(M0 | d) = BF10(d) P(M1)/(1-P(M1))

The posterior model probability for model M1 can be calculated as 

    P(M1 | d) = 1/(BF10(d) + (1-P(M1))/P(M1))

In addition, 

    P(M0 | d) = 1 - P(M1 | d) 

Does this help? 

Cheers,
Alexander

**Comment by fplatz** [comment]
Thanks Alexander for your help! However, I would appreciate to read those informations by means of an "easy-to-get-approach" ( :-) ). Even for teaching Bayes statistics for beginners it would make things more clear (and would support a step-by-step approach, "walking" from an a priori probability for H1 to the likelihood, the "evidence" and after all to H1's posterior probability). You're right, the calculations were possible in the way you have suggested, but it would be even faster for calculations of models' posterior probabilities if different a priori probabilities would have been selected (such as H1a: 20% instead of 50:50 for H1 and H0). From my perspective it might not be obvious enough for beginners.

Cheers,
Friedrich
